Forms Completion

RNtaate opened the pull request:
## What does this PR do?
- [X] Creates a User model.
- [X] Validates the username, email, and password for the users table.
- [X]  Creates a form template for creating a new user.
- [X] Creates a form template for editing an existing user.
- [X] Fixes rubocop and stylelint offenses.

maelfosso commented on app/views/users/new.html.erb:
```diff
@@ -0,0 +1,44 @@
+
+<% if false %>
+
+  <form accept-charset="UTF-8" action="/users" method="POST">
```
Please use `users_path` from rails helpers

maelfosso commented on app/views/users/new.html.erb:
```diff
@@ -0,0 +1,44 @@
+<% end %>
+
+<% if false%>
+  <%= form_tag("/users", method: "POST") do %>
```
Please use `users_path` from rails helpers

maelfosso commented on app/controllers/users_controller.rb:
```diff
@@ -0,0 +1,38 @@
+
+  def create
+    @user = User.new(user_params)
+    # @user = User.new(username: params[:username],
+    #                  email: params[:email],
+    #                  password: params[:password],
+    #                  password_confirmation: params[:password_confirmation])
```
Kindly remove comments

maelfosso reviewed (changes requested):
# Changes required :red_circle: 

Hello,

You did a good job
Kindly read the comments left below, make the suggested changes.
And submit a new code review request

Thanks

Mael FOSSO

RNtaate commented:
## Changes made
- [X] Changed action value in the first and second forms to users_path in new.html.erb file.
- [X] Removed unwanted comments from the users_controller.rb file.

christianotieno reviewed (approved):
# PROJECT APPROVED 🍾 
Excellent job on this project guys. :fireworks: Keep it up.
##
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job :+1:
- You can now merge these changes.
- I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
